### DISCUSSION TITLE
mac bigsur key board shortcuts have changed

### DISCUSSION THREAD

**Discussion by mickeyperlstein** [mdn/mdn-community#582] [Content]
In the [https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Search](https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Search) The shortcut is Shift-cmd-f, this is to make the app FULLSCREEN in bigsur. How do i do a full search using the mouse/menus etc?

**Comment by pepelsbey** [comment]
Closing as outdated/irrelevant.
